libs/wingc/examples/compile.rs

```diff
@@ -1,17 +1,19 @@
+// Compiles a source file and prints any errors to stderr.
```

Comment:
Do we really use this entrypoint? Maybe just get rid of it so we don't have to continue to maintain it...

Reply:
To support fast iteration cycles on development of wingc-specific features, it's super useful to have an entrypoint of some kind. The surface area is very small so I think it's worthwhile.

Reply:
OK